docker-entrypoint.sh

```diff
@@ -28,12 +29,16 @@ if [ "$1" == "kosmtik" ]; then
     sudo -H -u postgres kosmtik serve /style/output/OSMSmartrak/project.mml
 fi
 
+if [ -z "${S3_SUBDIRECTORY}" ]; then
+  STYLE_DIR="osm_tiles"
```

Comment:
```suggestion
  S3_SUBDIRECTORY="osm_tiles"
```

copypaste strikes again!

Reply:
Oops! Fixed.